pkg/instaclustr/mock/server/go/api_apache_kafka_connect_provisioning_v2_service.go

```diff
@@ -79,7 +87,9 @@ func (s *ApacheKafkaConnectProvisioningV2APIService) ClusterManagementV2Resource
 	// TODO - update ClusterManagementV2ResourcesApplicationsKafkaConnectClustersV2Post with the required logic for this service method.
 	// Add api_apache_kafka_connect_provisioning_v2_service.go to the .openapi-generator-ignore to avoid overwriting this service implementation when updating open api generation.
 
-	s.MockKafkaConnectCluster = &kafkaConnectClusterV2
-	s.MockKafkaConnectCluster.Id = CreatedID
-	return Response(202, s.MockKafkaConnectCluster), nil
+	id := uuid.New().String()
+	kafkaConnectClusterV2.Id = id
+	s.clusters[id] = &kafkaConnectClusterV2
```

Comment:
I think it's better to use the ID setting logic that I implemented in the stub server API service for the Cassandra resource.

Reply:
more control, imo

Reply:
It makes sense. I just forgot to revert these changes.